addons/l10n_mn/data/account.chart.template.csv

```diff
@@ -0,0 +1,2 @@
+id,property_account_receivable_id/id,property_account_payable_id/id,property_account_expense_categ_id/id,property_account_income_categ_id/id,income_currency_exchange_account_id/id,expense_currency_exchange_account_id/id
+l10n_mn.mn_chart_1,l10n_mn.account_template_1201_0201,l10n_mn.account_template_3101_0201,l10n_mn.account_template_7002_1101,l10n_mn.account_template_5101_0201,l10n_mn.account_template_5301_0201,l10n_mn.account_template_5302_0201
```

Comment:
Hello @jco-odoo 
Plz, Verify this default account template

Reply:
This is clearer to me in the xml.  Did you find this in the document?  (I can ask)

Reply:
No, I can't Find this in Documents.
But as per knowledge, I do this.
So need to Verify :smiley:

Reply:
Can you remove this csv and put it in the xml?  So you just need default account payable/receivable?